styles/base/typography.scss

```diff
@@ -1,5 +1,6 @@
 @import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=Roboto+Slab&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Source+Sans+3:ital,wght@0,200..900;1,200..900&display=swap');
```

Comment:
Slight oversight from the design handoff. It's probably better to just use one of the existing fonts in this project as loading multiple fonts can hurt load times a decent amount, especially for something so small

Reply:
Gotcha, I've removed the font import. It will default to sans-serif.